src/test/java/org/broadinstitute/consent/http/MockServerTestHelper.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.consent.http;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestPlan;
+import org.mockserver.client.MockServerClient;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class MockServerTestHelper extends AbstractTestHelper implements TestExecutionListener,
```

Comment:
Extends AbstractTestHelper so that a test that needs both the MockServerTestHelper and the AbstractTestHelper can easily extend a single class and be off to the races.